compiler/src/main/resources/non-compiling-programs/multi-state-declaration.cell

```diff
@@ -0,0 +1,9 @@
+world {
+    size = 100 [wrap], 200 [wrap]
+    tickrate = 1
+    cellsize = 5
+}
+
+state ident 5 (0, 0, 0) {
```

Comment:
Shouldn't there be square brackets around 5?

Reply:
That's not certain yet, as we don't have any grammar-rules on this. For some reason the program compiles when surrounded by square brackets and the AST does not show any information on this declaration.

So it's my opinion to just leave it as is.

Reply:
Let's revisit this later when their is more clarity around the syntax of multi-states

Reply:
Yes, exactly. So let's leave it as it - for now.